Assets/MixedRealityToolkit-Examples/UX/Scripts/LaunchDialogScript.cs

```diff
@@ -0,0 +1,63 @@
+﻿using MixedRealityToolkit.InputModule.EventData;
+using MixedRealityToolkit.UX.Dialog;
+using System.Collections;
+using UnityEngine;
+
+
+namespace MixedRealityToolkit.Examples.UX
+{
+    public class LaunchDialogScript : Interactive
+    {
+        public GameObject resultText;
+
+        [SerializeField]
+        private Dialog dialogPrefab;
+
+        public int numButtons;
```

Comment:
We should probably encapsulate the public fields and use proper PascalCasing for public and camelCasing for private.